admin/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "OSF_Admin",
```

Comment:
Same here: you should be able to use the NPM packages that are already installed.